spring-cloud-starter-stream-source-http/README.adoc

```diff
@@ -42,6 +42,8 @@ $$server.port$$:: $$Server HTTP port.$$ *($$Integer$$, default: `$$8080$$`)*
 
 NOTE: Security is disabled for this application by default.
 To enable it, you should use the mentioned above `http.enable-security = true` property.
+Also you must override the default `spring.autoconfigure.exclude` property by setting it to void (e.g. `--spring.autoconfigure.exclude=`)
```

Comment:
The real problem that we always have to specify this property even if we don't enable security.
That's indeed a bit awkward, but we don't have choice so far: to make this app working at all we always need to use this property.

We need to rephrase it somehow to reflect reality, maybe even point to the issue we are going to fix on the matter...